docs/content/docs/testing/telemetry.md

```diff
@@ -219,6 +212,9 @@ this.logger.sendTelemetryEvent({
 });
 ```
 
+Another point worth noting is that with the customized logics, it is a great place where you can send these events to
+your external telemetry system to have them logged.
```

Comment:
```suggestion
Because you can customize the logic, you can send these events to your own external telemetry system to have them logged. This enables you to integrate Fluid Framework logging with your other telemetry.
```